src/applications/static-pages/wizard/index.js

```diff
@@ -3,16 +3,40 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import set from 'platform/utilities/data/set';
 
+export const NO_BENEFIT_REFERRED = 'no benefit was referred';
+export const WIZARD_STATUS_NOT_STARTED = 'not started';
+export const WIZARD_STATUS_COMPLETE = 'complete';
+export const WIZARD_STATUS_APPLY_NOW = 'awaiting click on apply button';
+export const WIZARD_STATUS_IN_PROGRESS = 'in progress';
+export const WIZARD_STATUS_UPDATING = 'updating';
+export const formIdSuffixes = {
+  FORM_ID_1990: '1990',
+  FORM_ID_10203: '10203',
+  FORM_ID_1995: '1995',
+  FORM_ID_0994: '0994',
+  FORM_ID_5495: '5495',
+  FORM_ID_5490: '5490',
+  FORM_ID_1990E: '1990E',
+  FORM_ID_1990N: '1990N',
+};
```

Comment:
Should all these constants be added to this file? I'd think they are edu-benefits specific and should be included in that folder.

Reply:
Same idea for the `NO_BENEFIT_REFERRED` constant.

Reply:
Ok, I guess as long as @Clark-W-Griswold is cool with it 👍